luigi/target.py

```diff
@@ -316,18 +316,23 @@ class AtomicLocalFile(io.BufferedWriter):
     a temporary file in the local filesystem before
     moving it to its final destination.
 
-    This class is just for the writing part of the Target. See
-    :class:`luigi.file.LocalTarget` for example
+    This class handles only creation, cleanup of on-disk tmpfile.
+    Subclasses must implement `move_move_to_final_destination`.
+
+    :class:`luigi.local_target.atomic_file` example of local-endpoint atomic-file.
+    :class:`luigi.contrib.s3.AtomicS3File` example of remote-endpoint usage.
     """
 
     def __init__(self, path):
         self.__tmp_path = self.generate_tmp_path(path)
         self.path = path
+        self.unintended_close = False
```

Comment:
Make this private?

Reply:
Also this variable name is not descriptive I think. I don't understand what it does.